lib/celluloid.rb

```diff
@@ -478,17 +478,30 @@ def future(meth = nil, *args, &block)
 
 # Configure default systemwide settings
 
-Celluloid.task_class = begin
-  Celluloid.const_get(ENV['CELLULOID_TASK_CLASS'] || fail(TypeError))
-rescue
-  Celluloid::Task::Fibered
-end
 
-Celluloid.group_class = begin
-  Celluloid::Group.const_get(ENV['CELLULOID_GROUP_CLASS'] || fail(TypeError))
-rescue
-  Celluloid::Group::Spawner
-end
+Celluloid.task_class =
+  begin
+    str = ENV['CELLULOID_TASK_CLASS'] || 'Fibered'
+    Kernel.const_get(str)
```

Comment:
I wanted to *force* `Celluloid::Group` or `Celluloid::Task` as the base for either class type. Might seem petty, but this convention is a stern one. If it's really a hangup for people, they can override it in Ruby code directly.